core/tests/protractor_utils/action.js

```diff
@@ -76,9 +76,16 @@ var sendKeys = async function(
   await inputElement.sendKeys(keys);
 };
 
+var getText = async function(elementName, element) {
```

Comment:
Should we add this in sorted order (like earlier)? 

(Ditto below)

Reply:
done